torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -100,14 +101,24 @@ def special_pattern_replacement(model: QuantizedGraphModule) -> QuantizedGraphMo
     nodes = list(model.graph.nodes)
     for n in model.graph.nodes:
         q_node = n
-        if q_node.target == torch.quantize_per_tensor:
+        if q_node.target == torch.quantize_per_tensor or \
```

Comment:
optional: maybe we can do 

```
if not statement:
    continue
...logic...
```
to save some indent, here and also in L121?

Reply:
sg